packages/core/src/command/search-summary/filter.ts

```
@@ -0,0 +1,523 @@
+
+    const slimmerResource = removeUselessAttributes(resource);
+    const slimResource = applyResourceSpecificFilters(slimmerResource);
+
+    return slimResource ?? [];
+  });
+
+  return entries;
+}
+
+function removeUselessAttributes(res: Resource) {
+  delete res.meta;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)